notebook/converted/IRM_analysis.py

```diff
@@ -304,19 +323,26 @@ def eval_model(net, loader):
 print(label_list)
 
 
-# In[19]:
+# In[29]:
 
 
+held_out_platform_list = [plat.split('(')[-1].strip(')') for plat in held_out_platform_list]
 result_dict = {'f1_score': score_list, 'irm/erm': label_list, 'held_out_platform': held_out_platform_list}
 result_df = pd.DataFrame(result_dict)
 result_df.head()
 
 
-# In[20]:
+# ## Results
```

Comment:
Would it make sense to have the setup and training in a separate nb from the results? I know that for my work that once I've trained and tuned the model I can use it in multiple analyses so I'd rather not re-run the nb everytime from the beginning. 

I realize this is an older nb so feel free to make changes to this in another PR if you think thats appropriate.

Reply:
It's a good idea for the future, but this was a one-off analysis to have something to show in my prelim writeup.